### DISCUSSION TITLE
Vercel API delete domain

### DISCUSSION THREAD

**Discussion by hmjk99** [vercel/community#7236] [Help]
### Summary

When I try to delete a domain using the endpoint v6/domains/{name}, it is giving me a 403, forbidden error. But other endpoints like /v6/domains/${domain}/config or verify, or adding a domain are all working. But this delete is giving me a 403 error. I am also using the same token for all.

### Steps to Reproduce

<img width="534" alt="Screenshot 2024-06-19 at 1 05 15 AM" src="https://github.com/vercel/community/assets/124408498/72c54814-9a19-4eb9-beee-8ef435b6c151">

**Comment by versecafe** [comment]
@hmjk99 You can't delete a domain if there are any projects associated with/using it, you first have to remove all associations to it, then you can delete it.

**Reply by hmjk99** [reply]
@versecafe I delete the domain from the associated project before I make a delete request to /v6/domains/{domain name}. Then doesn't that mean there are no associations to it now?

[a comment by an automated account is not shown]
